fetcher-pipeline-api/app/main.py

```diff
@@ -0,0 +1,219 @@
+import glob
+import json
+import os
+import re
+from collections.abc import MutableSequence
+from hashlib import md5
+from mmap import mmap, ACCESS_READ
+from pathlib import Path
+from urllib.parse import urljoin
+
+from fastapi import FastAPI
+from pydantic import BaseModel
+
+from azure.servicebus.aio import ServiceBusClient
+from azure.servicebus import ServiceBusMessage
+
+
+import os
+
+NAMESPACE_CONNECTION_STR = os.getenv("NAMESPACE_CONNECTION_STR")
+QUEUE_NAME = os.getenv("QUEUE_NAME")
+CONTENT_URL = os.getenv("CONTENT_URL")
+CONTENT_DIR = os.getenv("CONTENT_DIR")
+
+# NAMESPACE_CONNECTION_STR = "connection_string"
+# QUEUE_NAME = "fetcher_queue"
+# CONTENT_URL = "https://audio-content.bibleineverylanguage.org"
+# CONTENT_DIR = "./content"
+
+
+class Filter(BaseModel):
+    language_id: str
+    resource_id: str
+    book_slug: str | None = None
+    chapter: int | None = None
+    exclude_format: MutableSequence[str] = []
+    exclude_quality: MutableSequence[str] = []
+    exclude_grouping: MutableSequence[str] = []
+
+
+class Parts(BaseModel):
+    language_id: str
+    resource_id: str
+    book_slug: str
+    chapter: int | None
+
+
+app = FastAPI()
+
+
+@app.get("/")
+def read_root():
+    return {"Hello": "World"}
+
+
+@app.get("/receive_messages")
```

Comment:
I would also remove this function. It meant to be created for tests. It not only gets messages from the service buss but also consumes them, so they are not anymore available for other functions.